modules/newFunc.js

```diff
@@ -0,0 +1,88 @@
+const tasksSection = document.querySelector('.task-section');
+const taskDesc = document.getElementById('task-value');
+
+let tasks = [
+  {
+    description: 'Create social Link',
+    completed: false,
+    index: 10,
+  }, {
+    description: 'Do Coding Exercise',
+    completed: false,
+    index: 11,
+  }, {
+    description: 'Doing physical Exercise',
+    completed: false,
+    index: 12,
+  },
```

Comment:
- [ ] As requested by the previous reviewer @topeogunleye you have hardcoded but it is not displaying on the browser as this is the `To Do list: list structure` project. So kindly fix it. The reviewer is in confusion as you made only one branch and add all the things and submitted the same pull request for all the projects. Kindly implement it said by @topeogunleye.